### PR TITLE
Add advisory note regarding update

### PR DESCRIPTION
**Pull request by ssh24**
Add docs for users to be aware of cloudant's behaviour on "update" of a document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,37 @@ User.destroyAll (function () {
 })
 ```
 
+#### Note:
+**Please be advised when using the Cloudant `updateOrCreate` functionality; Cloudant does not support the idea of updating a document. All "updates" on a document are _destructive_ replacements.**
```
Comment (jannyHou) [inline README.md]: `,` not `;`

File: README.md [hunk 2]
```diff
@@ -119,6 +119,37 @@ User.destroyAll (function () {
 })
 ```
 
+#### Note:
```
Comment (dhmlau) [inline README.md]: Just a sugggestion.... I'm ok either way. 
```
#### Note on using `updateOrCreate` functionality:
<remove the first phrase "please be advised..">Cloudant ..
```

File: README.md (hunk 1 above)
Comment (ssh24) [inline README.md]: It becomes a run-on sentence if we use `,` instead

Comment (0candy) [inline README.md]: How about `:`?

**Pull request merged**